content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Configuring Gateway Network Topology (Experimental)
+description: How to configure gateway network topology (experimental).
+weight: 60
+keywords: [traffic-management,ingress,gateway]
+---
+
+## (Experimental) Configuring network topologies
+
+Istio provides the ability to manage settings like X-Forwarded-For (XFF) and X-Forwarded-Client-Cert (XFCC), which are
+dependent on how the gateway workloads are deployed. This is currently an experimental feature.
+
+Many users choose to deploy Istio ingress gateways in using various network topologies
```

Comment:
Many users? How do you know that?

Instead, i'd go with

You may choose to deploy Istio ingress gateways using various network topologies

Reply:
Done